tests/auth_tests/test_tokens.py

```diff
@@ -32,6 +32,23 @@ def test_10265(self):
         tk2 = p0.make_token(reload)
         self.assertEqual(tk1, tk2)
 
+    def test_26615(self):
+        """
+        Ensure that the token generated for a user will fail if that user
+        changes email addresses.
+        """
+        # See ticket #26615
```

Comment:
That's what we keep in the commit message. `git blame` is your friend to find the commit and thus the commit message.